aspnetcore/mvc/views/razor.md

```diff
@@ -599,6 +630,46 @@ If multiple import files have the `@namespace` directive, the file closest to th
 
 The `@section` directive is used in conjunction with the [layout](xref:mvc/views/layout) to enable pages or views to render content in different parts of the HTML page. For more information, see [Sections](xref:mvc/views/layout#layout-sections-label).
 
+<a name="using"></a>
+
+### @using
+
+The `@using` directive adds the C# `using` directive to the generated view:
```

Comment:
If we're going to have a section on .razor files it would probably be valuable to state that this also controls what components are in scope.